hera_sim/visibilities/__init__.py

```diff
@@ -13,12 +13,19 @@
 # Registered Simulators
 from .vis_cpu import VisCPU
 
+
 try:
     from .healvis_wrapper import HealVis
-except ImportError:
+except (ImportError, NameError) as e: # NameError arises bc healvis is not Python 3 compliant
```

Comment:
Again, I don't think these are the types of errors we want to let pass silently. At the very least, we should be issuing a warning to the user that the simulator failed to be imported. On that note, I think it is worth investigating what is causing the import error. Finally, the note that "healvis is not Py3 compliant" is concerning; since Py2 is being deprecated at the end of the year, we are moving towards dropping Py2 support everywhere and making things compatible with only Py3. So I'm a little suspicious about this claim--it would be helpful to see the actual error message that pops up with the `NameError` so that we can debug.

Reply:
OK, I can work to make it a warning. 

This is the error in question (python 3 renamed `unicode` type to just `str`):
https://stackoverflow.com/questions/19877306/nameerror-global-name-unicode-is-not-defined-in-python-3

This `healvis` commit will fix the issue when the pull request for the `major-revisions` branch is approved:
https://github.com/RadioAstronomySoftwareGroup/healvis/pull/28/commits/4a2c2e90682b9831f66522e2c477d191a2ae3b8b

Reply:
Gotcha. In that case, I'd suggest minimal changes:
```
try:
    from .healvis_wrapper import HealVis
except ImportError, NameError:
    warnings.warn("healvis failed to import.")
```
or something along the lines of that. You'll need to `import warnings` as well (but do this with the rest of the imports).

Reply:
Done.

Reply:
Can we please instead make it:

```
try:
    from .healvis_wrapper import HealVis
except ImportError: # healvis not installed, that's fine
    HealVis = None
except NameError:
    warnings.warn("healvis failed to import, as it currently is only Python-2 compatible")
```